ts_benchmark/baselines/time_series_library/adapters_for_transformers.py

```diff
@@ -251,7 +252,11 @@ def forecast_fit(
 
         setattr(self.config, "task_name", "short_term_forecast")
         self.model = self.model_class(self.config)
-
+        
+        device_ids = np.arange(torch.cuda.device_count()).tolist()
```

Comment:
the backends makes use of `args.gpus` by setting `CUDA_VISIBLE_DEVICES`, so please use the devices specified in `CUDA_VISIBLE_DEVICES` to get the available devices here.